js/js-unit-testing/js-unit-testing_people/index.js

```diff
@@ -2,18 +2,54 @@ export function getFirstNames(people) {
   return people.map((person) => person.firstName);
 }
 
-export function getFullNames(people) {}
+export function getFullNames(people) {
+  return people.map((person) => `${person.firstName} ${person.lastName}`);
+}
+
+export function getNameAndAge(people) {
+  return people.map((person) => `${person.lastName} (${person.age})`);
+}
 
-export function getNameAndAge(people) {}
+export function getPeopleByAge(people, age) {
+  const peopleList = people.filter((person) => person.age >= age);
```

Comment:
here I think it is expected to get an array of objects (several persons out of people) I used the following function 
export function getPeopleByAge(people, age) {
  return people.filter((person) => person.age === age);